common/src/main/include/sensors/encoder.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <frc/CounterBase.h>
+
+namespace curtinfrc {
+  namespace sensors {
+    class encoder {
+     public:
+      encoder();
+      virtual double get_encoder_ticks();
```

Comment:
`virtual double get_encoder_ticks() = 0`, without the `0` it thinks you've got a .cpp file somewhere that defines this function.